src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -994,7 +994,7 @@ function MoneyRequestConfirmationList({
             shouldShowTextInput={false}
             shouldUseStyleForChildren={false}
             optionHoveredStyle={canModifyParticipants ? styles.hoveredComponentBG : {}}
-            footerContent={!isEditingSplitBill && footerContent}
+            footerContent={!isReadOnly && footerContent}
```

Comment:
So, in the previous PR, we merged 
https://github.com/Expensify/App/blob/fdca69cb3ae1cf73784e829333ae50dbfc71cb38/src/components/MoneyRequestConfirmationList.tsx#L599
and 
https://github.com/Expensify/App/blob/fdca69cb3ae1cf73784e829333ae50dbfc71cb38/src/components/MoneyTemporaryForRefactorRequestConfirmationList.tsx#L935
> [!NOTE]  
> `isEditingSplitBill` is `undefined` in `MoneyTemporaryForRefactorRequestConfirmationList`, so `!isEditingSplitBill` is always true


to
```js
footerContent={!isEditingSplitBill && footerContent}
```

@brunovjk Can you explain why this change fixes the issue?

Reply:
Sure! Just give me a moment, Im still on Android build :/
But to be honest I would like to hear your feedback on. Do you see a better alternative? Thanks.

Reply:
In the 'old' `MoneyRequestConfirmationList.tsx`, utilized in `SplitBillDetailsPage.tsx`, the footer is conditionally rendered based on `isEditingSplitBill`, in the `Scan` type you can edit. However, in `MoneyTemporaryForRefactorRequestConfirmationList`, as you mentioned, the footer is consistently rendered.

@eh2077 We need a new logic to approach both scenarios, I believe by utilizing `isReadOnly` we ensure that, what do you think? I've already tested it on the web and will upload the videos soon. My Android build is giving me troubles.

Reply:
I think we can go further. If we just do this:
```
footerContent={footerContent}
```
Since `footerContent` depends on `isReadOnly` anyway:

https://github.com/Expensify/App/blob/a04f40e09e6a933488957b7f6300edd6290c1ebd/src/components/MoneyRequestConfirmationList.tsx#L635-L638

But I'm not sure what problems this could cause.

Reply:
Cool, then I think we can go with `footerContent={footerContent}`

Reply:
Nice, let me do some testing